control-plane/pkg/reconciler/trigger/v2/triggerv2.go

```diff
@@ -118,7 +143,21 @@ func (r Reconciler) reconcileConsumerGroup(ctx context.Context, broker *eventing
 		offset = sources.OffsetEarliest
 	}
 
-	newcg := &internalscg.ConsumerGroup{
+	secret, err := security.Secret(ctx, &security.AnnotationsSecretLocator{Annotations: broker.Status.Annotations, Namespace: trigger.Namespace}, r.SecretProviderFunc())
+	if err != nil {
+		return nil, fmt.Errorf("failed to get secret: %w", err)
+	}
+	if err := r.TrackSecret(secret, broker); err != nil {
+		return nil, fmt.Errorf("failed to track secret: %w", err)
+	}
```

Comment:
Is the tracking needed here since we're not acting on the secret at all?

Also, if we need to track the secret, the parent object should be the one that we're reconciling, so in this case is trigger.

Reply:
You're right - we don't need to track it.